test/refda/test_eval.c

```diff
@@ -336,6 +400,10 @@ void test_refda_eval_target_valid(const char *targethex, const char *expectloghe
 
 // ari:/AC/(/BOOL/false,undefined,//1/1/OPER/bool-and) -> undefined
 TEST_CASE("8211838201F4F78401012568626F6F6C2D616E64", "F7")
+// ari:/AC/(/INT/0x0,/UVAST/1,//1/1/OPER/divide) -> undefined
```

Comment:
These are good to have verified here, are there other major conditions that will cause this output? What about trying integer/bitwise operators with at least one REAL32 input?

Reply:
Good call, there is another loose end / FIXME called out for REAL32. Let me look into that...

Reply:
OK, added a test case as well as the required real32 conversion function in `typing.c`.